src/gpuarray/buffer_blas.h

```diff
@@ -115,6 +115,30 @@ GPUARRAY_PUBLIC int gpublas_hgemmBatch(
   float beta, gpudata **C, size_t *offC, size_t ldc,
   size_t batchCount, int flags);
 
+GPUARRAY_PUBLIC int gpublas_hgemm3D(
```

Comment:
There is still the question of the fct name. Why not use the cublas name? Also, I think this 3d can be confonding, as what is a 3d gemm? People could think about tensordot. So there is many interpretation, that isn't good.